internal/services/systemcentervirtualmachinemanager/system_center_virtual_machine_manager_inventory_items_data_source.go

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package systemcentervirtualmachinemanager
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/inventoryitems"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/vmmservers"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type SystemCenterVirtualMachineManagerInventoryItemsDataSource struct{}
+
+var _ sdk.DataSource = SystemCenterVirtualMachineManagerInventoryItemsDataSource{}
+
+type SystemCenterVirtualMachineManagerInventoryItemsDataSourceModel struct {
+	SystemCenterVirtualMachineManagerServerId string          `tfschema:"system_center_virtual_machine_manager_server_id"`
+	InventoryItems                            []InventoryItem `tfschema:"inventory_items"`
+}
+
+type InventoryItem struct {
+	id            string `tfschema:"id"`
+	name          string `tfschema:"name"`
```

Comment:
neither of these two fields is exported, so would need to be to be set into the state

Reply:
Seems I exposed them.
[New Data Source: `azurerm_system_center_virtual_machine_manager_inventory_items` by neil-yechenwei · Pull Request #25110 · hashicorp/terraform-provider-azurerm (github.com)](https://github.com/hashicorp/terraform-provider-azurerm/pull/25110/files#diff-c5baf72d025eba16a924fd648f2491926835bf247f2d63c86d3f63af8cfee713R132)
[New Data Source: `azurerm_system_center_virtual_machine_manager_inventory_items` by neil-yechenwei · Pull Request #25110 · hashicorp/terraform-provider-azurerm (github.com)](https://github.com/hashicorp/terraform-provider-azurerm/pull/25110/files#diff-c5baf72d025eba16a924fd648f2491926835bf247f2d63c86d3f63af8cfee713R137)